SST and SA options (#75)
su2code/su2code.github.io: _docs_v7/Physical-Definition.md

## Flow Condition (Incompressible) ##

| Solver | Version |
| --- | --- |

> | `INC_EULER`, `INC_NAVIER_STOKES`, `INC_RANS` | 7.0.0 |

Reviewer: Do these 7.0.0 need to be updated?

Author: The info there is still valid for v7, what is missing here that I noticed are the NEMO solvers

**Note:** The initial state is also used as boundary conditions for `MARKER_FAR`.

## Turbulence Models ##

| Solver | Version |
| --- | --- |
| `*_RANS` | 7.4.0 |

> This section describes how to setup turbulence models for RANS simulations. In general turbulence models are selected via the option `KIND_TURB_MODEL`, corrections/versions and parameters of the models are specified via the options listed below.
> The turbulent Pradtl number can be modified with option `PRANDTL_TURB` (the default is 0.9).

Reviewer 2: ```suggestion
This section describes how to setup turbulence models for RANS simulations. The turbulence model is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`
A turbulence model can then be selected via the option `KIND_TURB_MODEL`
Different submodels and parameters are specified via the different options listed below.
The turbulent Prandtl number can be modified with the option `PRANDTL_TURB` (the default is 0.9).
```

Reviewer 2: The "Prandtl" remark is a bit out of place... maybe first mention all KIND_TURB_MODEL and XXX_OPTIONS and then other shared options for turbulence?

### Spalart-Allmaras (SA) ###

SU2 implements several versions and corrections of the SA model.

> The model is selected using `KIND_TURB_MODEL= SA` and the modifications via the `SA_OPTIONS` list, if this list is empty SU2 defaults to `SA-noft2`.

Reviewer 2: ```suggestion
The model is selected using `KIND_TURB_MODEL= SA` and the modifications via the `SA_OPTIONS` list. If this list is empty, then SU2 defaults to `SA-noft2`.
```

- Versions:
  - `NEGATIVE` - Negative SA model.
  - `EDWARDS` - Edwards modification.
  - `BCM` - BCM transitional model.
  - `FT2` - SA model **with** ft2 term, note that by default we omit this term.
- Corrections:
  - `QCR2000` - Quadratic contitutive relation used in the stress tensor.
  - `COMPRESSIBILITY` - Mixing layer compressibility correction.
  - `ROTATION` - Dacles-Mariani et al. rotation correction.

All the modifications can be combined with each other expect `NEGATIVE`, `EDWARDS`, and `BCM`. However some combinations are not considered standard, e.g. `SA-neg-noft2`, see TMR for details.
For example, to specify `SA-neg-R-comp-QCR2000` use `SA_OPTIONS= NEGATIVE, FT2, ROTATION, COMPRESSIBILITY, QCR2000`.

The rough wall correction is implicitly turned on by specifying roughness values for wall markers via the `WALL_ROUGHNESS` option.

### Shear Stress Transport (SST) ###

SU2 implements the "Standard" (1994) and 2003 versions of the SST model along with several modifications.

**Note:** Currently all versions are "modified" i.e. the turbulence kinetic energy (TKE) is not included in the viscous stress tensor.

> The model is selected using `KIND_TURB_MODEL= SST` and the modifications via the `SST_OPTIONS` list, if this list is empty SU2 defaults to `SSTm` (see warning below).

Reviewer 2: ```suggestion
The main model is selected using `KIND_TURB_MODEL= SST` and the version and modifications via the `SST_OPTIONS` list. If this list is empty SU2 defaults to the baseline 1994 model, `SSTm` (see warning below).
```

**Note:** The initial state is also used as boundary conditions for `MARKER_FAR`.

## Turbulence Models ##

| Solver | Version |
| --- | --- |
| `*_RANS` | 7.4.0 |

> This section describes how to setup turbulence models for RANS simulations. The turbulence model is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`

Author: ```suggestion
This section describes how to setup turbulence models for RANS simulations. Turbulence is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`
```
